src/Console/ProjectionRebuildCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Console;
+
+use Patchlevel\EventSourcing\Pipeline\Pipeline;
+use Patchlevel\EventSourcing\Pipeline\Source\StreamableStoreSource;
+use Patchlevel\EventSourcing\Pipeline\Target\ProjectionRepositoryTarget;
+use Patchlevel\EventSourcing\Projection\ProjectionRepository;
+use Patchlevel\EventSourcing\Store\Store;
+use Patchlevel\EventSourcing\Store\StreamableStore;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ProjectionRebuildCommand extends Command
+{
+    private Store $store;
+    private ProjectionRepository $projectionRepository;
+
+    public function __construct(Store $store, ProjectionRepository $projectionRepository)
```

Comment:
Since we are only supporting `StreamableStore` we could typhint it right here instead of duck type checking it at the execution

Reply:
lets investigate this again some time later